### PR TITLE
Add script that generate encoded DOCKER_CONFIG for Openshift deployment

### PR DESCRIPTION
**Pull request by Dysproz**
This short script generates DOCKER_CONFIG which should be provided in config to install-manifests for Openshift

### CODE REVIEW

File: deploy/openshift/tools/docker-config-generate/generate-docker-config.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```
Comment (kaweue) [inline deploy/openshift/tools/docker-config-generate/generate-docker-config.sh:1]: Why not #!/usr/bin/env bash?

Comment (Dysproz) [reply deploy/openshift/tools/docker-config-generate/generate-docker-config.sh]: Done

**Comment by psykulsk** [comment]
recheck

**Comment by Dysproz** [comment]
recheck ha

**Comment by Dysproz** [comment]
recheck ha

File: deploy/openshift/tools/docker-config-generate/generate-docker-config.sh [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    read -p "Docker registry: "  REGISTRY
+    read -p "Docker username: "  USERNAME
+    read -s -p "Docker secret: "  SECRET
+}
+
+generate_base64() {
+    AUTH=$(echo $USERNAME:$SECRET | base64 -w 0)
+    RENDERED_JSON=$(echo {\"auths\":{\"${REGISTRY}\":{\"username\": \"${USERNAME}\",\"password\":\"${SECRET}\",\"auth\":\"${AUTH}\"}}})
+    DOCKER_CONFIG=$(echo $RENDERED_JSON | base64 -w 0)
+}
```
Comment (psykulsk) [inline deploy/openshift/tools/docker-config-generate/generate-docker-config.sh:13]: Unfortunately base64 on macos does not have the -w option. And most of the people that are going to be using this, are using macos. I've seen suggestions to use base64 encoder built in openssl (e.g. `openssl base64`) that works fine on most OS's and doesn't add new lines.

Comment (psykulsk) [reply deploy/openshift/tools/docker-config-generate/generate-docker-config.sh:13]: So I would try changing `base64 -w 0` to `openssl base64` and adding a comment about that it is used for compatibility reasons.

Comment (Dysproz) [reply deploy/openshift/tools/docker-config-generate/generate-docker-config.sh]: Done

**Comment by Dysproz** [comment]
recheck

**Pull request merged**